Adds 2021-04-07-cultural-competency-in-archives-phase-one-of-the-education-campaign (#115)
RockefellerArchiveCenter/bits_and_bytes: _posts/2021-04-07-cultural-competency-in-archives-phase-one-of-the-education-campaign.md

---
title: "Learning Together: Cultural Competency in Archives: Phase One of the Education Campaign"
date: 2021-04-07T11:30:00
author: Katie Martin
layout: post
categories:
  - Collaboration
  - Conferences/Education
tags:
  - cultural competency
  - description
  - learning
excerpt_separator: <!--more-->
---

> The Rockefeller Archive Center is taking steps to ensure that our actions align with the RAC [statement in solidarity with the Black Community](https://twitter.com/rockarch_org/status/1269009567935062016?s=20) and our recent [statement in solidarity with the Asian, Asian-American and Pacific Islander Community](https://twitter.com/rockarch_org/status/1374083568939450368?s=20). Amy Berish, Darren Young, and myself, on behalf of the processing team, [organized an education campaign](https://blog.rockarch.org/cultural-competency-in-archives-planning-an-education-campaign) to learn more about cultural competency concepts and ultimately create more inclusive and respectful archival description.

Reviewer: ```suggestion
The Rockefeller Archive Center is taking steps to ensure that our actions align with the RAC [statement in solidarity with the Black Community](https://twitter.com/rockarch_org/status/1269009567935062016?s=20) and our recent [statement in solidarity with the Asian, Asian-American and Pacific Islander Community](https://twitter.com/rockarch_org/status/1374083568939450368?s=20). Amy Berish, Darren Young, and myself, on behalf of the Processing team, [organized an education campaign](https://blog.rockarch.org/cultural-competency-in-archives-planning-an-education-campaign) to learn more about cultural competency concepts and ultimately create more inclusive and respectful archival description.
```

> The Rockefeller Archive Center is taking steps to ensure that our actions align with the RAC [statement in solidarity with the Black Community](https://twitter.com/rockarch_org/status/1269009567935062016?s=20) and our recent [statement in solidarity with the Asian, Asian-American and Pacific Islander Community](https://twitter.com/rockarch_org/status/1374083568939450368?s=20). Amy Berish, Darren Young, and myself, on behalf of the processing team, [organized an education campaign](https://blog.rockarch.org/cultural-competency-in-archives-planning-an-education-campaign) to learn more about cultural competency concepts and ultimately create more inclusive and respectful archival description.

Reviewer 2: ```suggestion
The Rockefeller Archive Center is taking steps to ensure that our actions align with the RAC [statement in solidarity with the Black Community](https://twitter.com/rockarch_org/status/1269009567935062016?s=20) and our recent [statement in solidarity with the Asian, Asian-American and Pacific Islander Community](https://twitter.com/rockarch_org/status/1374083568939450368?s=20). Amy Berish, Darren Young, and myself, on behalf of the processing team, [organized an education campaign](https://blog.rockarch.org/cultural-competency-in-archives-planning-an-education-campaign) to help us and our colleagues learn more about cultural competency concepts and ultimately create more inclusive and respectful archival description.
```

Reviewer 2: The wording might not be quite right, but I think it makes sense to point out that this was for everyone, not just the three of you.

- LinkedIn Learning [Confronting Racism Course](https://www.linkedin.com/learning/confronting-racism-with-robin-diangelo)
- SAA [Cultural Diversity Competency Course](https://www2.archivists.org/prof-education/course-catalog/cultural-diversity-competency)
- SAA Archives*Records 2020, On-Demand Session 24: [Implementing Inclusive (Re)Description at Predominantly White Institutions](https://mvp.markeys.onl/SAA/agenda/session/306593)
- Three discussion sessions in January 2021 to accommodate staff

Early in the planning process, we acknowledged that the nature of these discussions would sometimes be sensitive, challenging, and require personal reflection. To make the sessions more transparent, we supplied the agenda and our slides beforehand.  We also sent out an anonymous pre-discussion survey to provide more avenues for participation outside the sessions. The survey helped us determine how to structure the sessions and gauge reactions to the learning modules and the education campaign itself.

*Pre-Discussion Survey Questions:*

> For many, race and racism are uncomfortable topics to discuss. What makes these topics uncomfortable for you, personally? How have these learning modules impacted your ability to engage with conversations around race and racism?

Reviewer: ```suggestion
- For many, race and racism are uncomfortable topics to discuss. What makes these topics uncomfortable for you, personally? How have these learning modules impacted your ability to engage with conversations around race and racism?
```

- What activities, discussion questions, and/or exercises presented within the learning modules did you find the most engaging and/or challenging?

## Discussion Sessions

While Amy, Darren, and I organized and facilitated this phase of the education campaign, it is important to note that we were also learning.  We took the courses, answered the pre-survey questions, and shared our thoughts as part of the sessions. Our experience with the learning modules guided the way we structured the sessions in ways that encouraged an inclusive and open dialogue.  At the start of each session, we provided [ground rules](https://extension.umn.edu/public-engagement-strategies/setting-ground-rules-productive-discussions) based on guidance from the University of Minnesota, for example: seek first to understand, not to be understood, and criticize the discussion, not the person. Prior to the session,  we asked that everyone prepare an answer to an icebreaker question: “What was your biggest takeaway from the learning modules?” We wanted to provide everyone the opportunity to express their views on the modules and ensure that all voices were heard.

> After the initial icebreaker, we moved into breakout rooms for a larger conversation about the Rockefeller Archive Center itself.  Because we encouraged all staff to participate, we understood that there might be discomfort expressing certain views in front of higher-ups and teammates. To encourage more free discussion, we created breakout rooms groups that varied across the hierarchy of the organization and across departments. The main discussion consisted of three broad topics.

Reviewer: ```suggestion
After the initial icebreaker, we moved into breakout rooms for a larger conversation about the Rockefeller Archive Center itself.  Because we encouraged all staff to participate, we understood that there might be discomfort expressing certain views in front of managers and teammates. To encourage more free discussion, we created breakout rooms groups that varied across the hierarchy of the organization and across departments. The main discussion consisted of three broad topics.
```

*Topics for Discussion*

- Workplace Culture at the  RAC - reactions to answers from the pre-survey
- RAC as an Institution - How can we take first impressions into account when greeting researchers? How we might contribute to [gatekeeping](https://blogs.ifla.org/faife/2019/07/17/from-gatekeeper-to-gateway-to-gate-opener-the-changing-role-of-libraries-and-how-we-talk-about-it/)?
- Whiteness at the Rockefeller Archive Center - How might the demographic makeup of our records’ creators impact the way researchers, staff members, and others perceive the RAC? How may the ways in which the creators perceive and describe the subjects differ from the ways in which the subjects perceive and describe themselves?

> In total, the discussion sessions were scheduled for an hour and a half each. To manage time, we tried to keep the icebreaker around 20 minutes with 45 minutes left for the three discussion sections and 10 minutes at the end to report back from breakout rooms. Following the sessions, we sent out a post-survey to inform how we move forward into a monthly reading group for phase two of the education campaign.

Reviewer: like the pre-survey, it may be useful to note that the post-survey was also anonymous.
